src/status_im/ui/screens/contacts/contact_list_modal/views.cljs

```diff
@@ -61,20 +62,21 @@
             params [:get :contacts/click-params]]
     [drawer-view
      [view {:flex 1}
-      [status-bar {:type :modal}]
+      [status-bar]
       [contact-list-modal-toolbar]
       [list-view {:dataSource                (lw/to-datasource contacts)
```

Comment:
It could be an opportunity to replace with the new `list` component which is simpler to use and more performant.

Reply:
Maybe we could have separate PR for this? to replace all lists, not sure if i should do it only for this list in this PR, because as i understand we don't have new list

Reply:
Sure! I will create a dedicated ticket for the migration.

The list I was referring to is `component.list`, especially `flat-list`.

Reply:
wow, cool, missed it somehow, will implement then

Reply:
we have `:renderHeader` for supporting old dapp wallet, so better to replace this list when we remove dapp wallet

Reply:
:+1: